src/pages/home/GoogleOneTap.ts

```diff
@@ -0,0 +1,166 @@
+import { AntelopeError } from 'src/antelope/types';
+
+import * as Buffer from 'buffer';
+import { BehaviorSubject, Subject } from 'rxjs';
+
```

Comment:
This is an auxiliary file with useful functions to handle everything related to the GoogleOneTap service.